qdrant-landing/content/documentation/private-cloud/api-reference.md

```diff
@@ -57,6 +59,26 @@ _Appears in:_
 
 
 
+#### ComponentReference
+
+
+
+
+
+
```

Comment:
are so many new lines required?

Reply:
The come from the generated docs and will be collapsed automatically once transformed to HTML